admin-dev/themes/new-theme/js/pages/product/edit/product-seo-manager.ts

```diff
@@ -74,5 +75,14 @@ export default class ProductSEOManager {
       },
       previewUrl,
     );
+
+    new TaggableField({
+      tokenFieldSelector: ProductMap.seo.tagFields,
+      options: {
+        createTokensOnBlur: true,
+        delimiter: ',',
+        minWidth: '768px',
```

Comment:
Damn, did I miss this somewhere? We need to make sure this will work on mobile

Reply:
Bootstrap component doesn't allow to prevent this auto computing and I couldn't find a better solution If you have one be my guest